package spob;

message RecoverCommit {
  required uint32 primary = 1;
};